test/unit/adapters/sequelize.test.js

```diff
@@ -836,6 +837,12 @@ describe('model.init with getterMethods and setterMethods', () => {
     }
   });
 
+  // multiple implement
+  class UU extends User {
+    get j () {
+      return 'j';
+    }
+  }
```

Comment:
`class CustomUser` might be preferred. If this fixture is only used in the `toJSON()` test case, should've considered putting it within that test. Or better create another specific test describing intended test purpose.